pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -53,6 +54,15 @@ const (
 	// workload-bound annotations for controlling mixerfilter
 	policyCheckAnnotation = "policy.istio.io/check"
 
+	// workload-bound annotation for max retries on transport error
+	policyCheckRetriesAnnotation = "policy.istio.io/checkRetries"
+
+	// workload-bound annotation for base time to wait between retries
```

Comment:
Not sure a user would understand from comment or name. Maybe few more words, example value, etc ?

Reply:
And format - duration is not what most users expects.